source/documentation/other-topics/maintenance-page.md

```diff
@@ -0,0 +1,21 @@
+## Applying a Maintenance Page
+
+### Overview
+This document will serve as a guide on how to apply a default maintenance page to your application on the Cloud Platform.
+
+### Deploying the page 
+A repository has been created to store all the files related to the maintenance page: [cloud-platform-maintenance-page](https://github.com/ministryofjustice/cloud-platform-maintenance-page)
+
+The repository contains the manifest files needed to deploy a standard maintenance page into your namespace. This directory also contains the maintenance page HTML file, along with a DockerFile to compile an image.
+
+Within the `Kubectl_deploy` directory, there are 3 simple manifest files that make up the deployment, `deploy.yaml`, `ingress.yaml` and `service.yaml`.
+
+You must remove your applications current ingress file from the namespace before applying this page.
```

Comment:
If they remove their ingress and apply for that repo, it will create an `Ingress` with the "wrong" hostname. Should we instead remove the `Ingress` definition from that repo and ask users to:
1. deploy using those files (ie. create `Deployment` & `Service`)
2. update the `backend` on their *existing* `Ingress` to point to the maintenance page `Service`?